metis-indexing/src/main/java/eu/europeana/indexing/fullbean/EuropeanaAggregationFieldInput.java

```diff
@@ -1,14 +1,12 @@
 package eu.europeana.indexing.fullbean;
```

Comment:
Is then the field qualityAnnotations required?
Can we not remove it from the class now?

Reply:
true. is not required here any more. removed.